--- src/matrix/kaldi-blas.h.orig	2014-01-30 10:06:38.440313000 +0800
+++ src/matrix/kaldi-blas.h	2014-01-30 09:07:20.573264000 +0800
@@ -88,7 +88,7 @@
     #include <mkl.h>
   }
 #elif defined(HAVE_OPENBLAS)
-extern "C" {
+//extern "C" {
   // getting cblas.h and lapacke.h from <openblas-install-dir>/.
   // putting in "" not <> to search -I before system libraries.
   #include "cblas.h"
@@ -105,7 +105,7 @@
   #undef bit_test
   #undef bit_clear
   #undef bit_set
-}
+//}
 #else
   #error "You need to define (using the preprocessor) either HAVE_CLAPACK or HAVE_ATLAS or HAVE_MKL (but not more than one)"  
 #endif
